### PR TITLE
Calculator Styling (Page)

### LINKED ISSUE #7: Creating calculator component
a component that renders  the activities with their symbols/icons with their own styling and own **onClick function** ( if it's a time or amount, but if we are doing by 1 minutes and 1 times for each click then the event listeners functions should be the same ).

also this component has a `Calculate` button which calculates what the user has chosen with the data that we have using the calculate module #14 , and renders back the overall cost and overall liters.

### Work need to be done:

- [x] Functionality 
   - [x] Clickable icons
   - [x] Stacking +1 Functionality
   - [x] Working Calculator (X Liters = x Shekels)
    - [x] :point_right:  The calculator should reset everything :point_left: :exclamation:

- [x] Design 
  - [X] Icons
  - [x] Dekstop Design
  - [x] Mobile Design
  - [x] Calculate Button

### PR DESCRIPTION
**Pull request by Amirk390**
Created calculator page with a component

close #7

### CODE REVIEW

**Review by SamiSha99 — APPROVED** [review]
Looks really good to me :white_check_mark:

File: src/Calculator.jsx [hunk 1]
```diff
@@ -4,81 +4,185 @@ import styled from 'styled-components';
...
   <Container>
     <SubTitle>
       Please select your daily activity and choose the amout by clicking on it again
     </SubTitle>
-    <Content>
+    <Activitys>
       <IconContainer>
         <Icon>
-          <i className="fa fa-bath" />
+          <Img src="/bottle.png" alt="" />
```
Comment (SamiSha99) [inline src/Calculator.jsx]: You shouldn't keep ```alt=""```, if you aren't going to add anything at least remove them, do the following to all  elements.

**Review by SamiSha99 — CHANGES_REQUESTED** [review]
Small change

File: src/Calculator.jsx (hunk 1 above)
Comment (Amirk390) [reply src/Calculator.jsx]: Good catch Sami

**Review by SamiSha99 — APPROVED** [review]
All good now :white_check_mark:

File: src/Calculator.jsx [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+      </IconContainer>
+      <Calculate>CALCULATE</Calculate>
+    </Activitys>
+    <ResultsContainer>
+      <Results>
+        <ResultsTitle>Your cost</ResultsTitle>
+        <Numbers>5</Numbers>
+      </Results>
+      <Results>
+        <ResultsTitle>LITERS</ResultsTitle>
```
Comment (matthewdking) [inline src/Calculator.jsx]: Is there a reason this is in capitals and your cost is not? Be consistent

File: src/NavBar.jsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ const MenuContainer = styled.div`
 
 const Nav = styled.nav`
   display: flex;
-  max-height: 3rem;
+  max-height: 2rem;
```
Comment (matthewdking) [inline src/NavBar.jsx:15]: This keeps changing every PR :laughing:

File: src/Calculator.jsx [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+  width: 2rem;
+  height: 2rem;
+`;
+
+const Calculate = styled.button`
+  display: flex;
+  justify-content: center;
+  font-size: 1rem;
+  font-weight: bold;
+  color: #383854;
```
Comment (matthewdking) [inline src/Calculator.jsx]: I have already given my opinion on this button being a different colour! It is a call to action and I think having a more stand out colour makes sense here. The [google material design docs](https://material.io/guidelines/) are useful as they lay out some styling guidelines.

**Review by matthewdking — CHANGES_REQUESTED** [review]
This issue addresses #7 which mentions the ability to click on buttons? None of this has been implemented? Will this come in a future PR? If yes then the issues need updating to address this because this PR doesn't complete the brief of the issue.

@Karyum @Amirk390 @SamiSha99

**Comment by Karyum** [comment]
@matthewdking yeah good poing we should split the issues into styling/html and functionality.

@Amirk390 or @SamiSha99 can you please sort this out by specifying in the issue the we split the two and make checkboxes for them

File: src/Calculator.jsx (hunk 4 above)
Comment (Amirk390) [reply src/Calculator.jsx]: i'll work on that

File: src/NavBar.jsx (hunk 3 above)
Comment (Amirk390) [reply src/NavBar.jsx:15]: Mario changed that :stuck_out_tongue_closed_eyes:

**Comment by Amirk390** [comment]
@Karyum 
i agree on your comment, i updated the related issue and im changing this PR title to 
Calculator Styling (Page)

**Review by SamiSha99 — APPROVED** [review]
:white_check_mark:

File: src/Calculator.jsx [hunk 5]
```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.section`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  color: #383854;
+  margin: 2rem 0 2rem 0;
```
Comment (matthewdking) [inline src/Calculator.jsx]: this can be shortened to
```
margin: 2rem 0;
```

File: src/Calculator.jsx [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+
+  @media only screen and (max-width: 420px) {
+    padding: 0;
+  }
+`;
+
+const IconContainer = styled.section`
+  display: flex;
+  justify-content: center;
+  margin: 1rem 0 0 0;
```
Comment (matthewdking) [inline src/Calculator.jsx]: just do `margin-top: 1rem` it is just confusing having a load of zeroes after

File: src/Calculator.jsx [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+  border-radius: 1rem;
+  color: white;
+  background-color: #383854;
+  border: 0.05rem rgba(255, 255, 255, 0.75) solid;
+  box-shadow: 0.2rem 0.2rem 0.3rem #888888;
+  margin: 0 0.5rem;
+  padding: 2rem 2rem;
+
+  @media only screen and (max-width: 700px) {
+    padding: 1rem 1rem;
```
Comment (matthewdking) [inline src/Calculator.jsx]: `padding: 1rem`

File: src/Calculator.jsx (hunk 4 above)
Comment (matthewdking) [reply src/Calculator.jsx]: Personal preference I will let this one go

File: src/NavBar.jsx (hunk 3 above)
Comment (matthewdking) [reply src/NavBar.jsx:15]: It also seems odd that the Navbar is being styled in this PR?!?!

**Review by matthewdking — CHANGES_REQUESTED** [review]
A few small changes

File: src/NavBar.jsx (hunk 3 above)
Comment (Amirk390) [reply src/NavBar.jsx:15]: Yeah, while working on the design the navbar was too big so we modified it a bit

**Comment by Amirk390** [comment]
@SamiSha99 Please review this version

**Review by SamiSha99 — APPROVED** [review]
Looks good for me, can you also please @Amirk390 review the calculator-function PR?

**Comment by Amirk390** [comment]
@SamiSha99
> Looks good for me, can you also please @Amirk390 review the calculator-function PR?

i did

**Review by matthewdking — APPROVED** [review]

**Pull request merged**